examples/webgl_shaders_sky.html

```diff
@@ -75,7 +75,7 @@
 				gui.add( effectController, "rayleigh", 0.0, 4, 0.001 ).onChange( guiChanged );
 				gui.add( effectController, "mieCoefficient", 0.0, 0.1, 0.001 ).onChange( guiChanged );
 				gui.add( effectController, "mieDirectionalG", 0.0, 1, 0.001 ).onChange( guiChanged );
-				gui.add( effectController, "inclination", 0, 1, 0.0001 ).onChange( guiChanged );
+				gui.add( effectController, "inclination", -1, 1, 0.0001 ).onChange( guiChanged );
```

Comment:
Does a negative `inclination` make sense? I think the range should be like in `webgl_shaders_ocean`.

Reply:
I think the proper term is solar _elevation_. (The term solar _altitude_ is also used, but that can be confusing for obvious reasons.)

Solar elevation is in degrees, and can be negative (at twilight, for example). A proper model should handle that case.

Solar azimuth is also in degrees.

/ping @zz85

Reply:
> Does a negative inclination make sense?

I'm asking this because changing to a negative value has no visual effect in the demo.

Reply:
Any value smaller than -0.0257 has no visual effect, to be more precise. Values lower than that may make no difference visually, but it does allow the user to see the effect of when the sun is at the opposite side of the world.

Reply:
https://en.wikipedia.org/wiki/Twilight

Reply:
@sirxemic Then it's best to update the line in `webgl_shaders_ocean`.

Reply:
2aad3193a80bf205b6bb687daa0147d26c1924e8

Reply:
As I suggested, I think it is best to use the term elevation, and set units of elevation and azimuth in degrees.

Reply:
@sirxemic Are you going to make this change? Otherwise I can jump in so we get this PR finally merged^^.

Reply:
The fact that scene goes dark when the sun hits the horizon is either an error in the model, or an error in the implementation of it. That can be saved for another time if someone is interested in pursuing it.

Reply:
@WestLangley What range of angles would you suggest for `azimuth` and `elevation`. Looking at the Wikipedia article it seems the angles are always positive.

Assuming we would have `[-90, 90]` for `elevation`. What part of the hemisphere would be expressed in negative angles?

Reply:
@Mugen87 

From https://www.esrl.noaa.gov/gmd/grad/solcalc/azel.html

>Azimuth is measured in degrees clockwise from north. Elevation is measured in degrees up from the horizon.

I think it is OK for azimuth to be in `[ -180, 180 ]` for the demo, or `[ 0, 360 ]` -- whatever you prefer. We do not have a concept of 'north'.

Since the model goes black below the horizon, `[ 0, 90 ]` for elevation in the demo seems appropriate to me. (Negative values, if supported by the model, would correspond to pre-dawn or post-sunset, when the sun is just below the horizon.

JMHO.   :-)

Reply:
thanks for spotting this, and for interesting discussion. agreed with the points @WestLangley brought up - azimuth and elevation make sense, and so is having some notion of sun below the horizon for twilight. the model used in the shader have some limited support for negative elevation, which have already been pointed up here :)

Reply:
>the model used in the shader have some limited support for negative elevation

Hi @zz85! Very limited -- and not to [this extent](https://en.wikipedia.org/wiki/Twilight#/media/File:Twilight-dawn_subcategories.svg), I think.